samples/graph-appcatalog-lifecycle/nodejs/README.md

```diff
@@ -0,0 +1,88 @@
+﻿### AppCatalog sample
+
+Bot Framework v4 AppCatalog sample.
+
+This sample shows a AppCatalog bot and demonstrates teamsApp lifecycle in catalog followed by commands given to Bot.
+
+## Prerequisites
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+  B) Or from Visual Studio code
+
+  - Launch Visual Studio code
+  - File -> Open Folder
+  - Navigate to `graph-appcatalog-lifecycle/node` folder
+  - Press `F5` to run the project
+## Instruction on setting connection string for bot authentication on the behalf of user
+1. In the Azure portal, select your resource group from the dashboard.
+
+2. Select your bot channel registration link.
+
+3. Open the resource page and select Configuration under Settings.
+
+4. Select Add OAuth Connection Settings.
+[Provide the connectionstring1 image url here]
+5. Complete the form as follows:
+
+a. Name. Enter a name for the connection. You'll use this name in your bot in the appsettings.json file. For example BotTeamsAuthADv1.
+
+b. Service Provider. Select Azure Active Directory. Once you select this, the Azure AD-specific fields will be displayed.
+
+c. Client id. Enter the Application (client) ID that you recorded for your Azure identity provider app in the steps above.
+
+d. Client secret. Enter the secret that you recorded for your Azure identity provider app in the steps above.
+
+e. Grant Type. Enter authorization_code.
+
+f. Login URL. Enter https://login.microsoftonline.com.
+
+g. Tenant ID, enter the Directory (tenant) ID that you recorded earlier for your Azure identity app or common depending on the supported account type selected when you created the identity provider app.
+h. For Resource URL, enter https://graph.microsoft.com/
+i. Provide  Scopes like "AppCatalog.Submit, AppCatalog.Read.All, AppCatalog.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All"
+[Provide the connectionstring2 image url here]
+A) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http -host-header=rewrite 3978
+## Concepts introduced in this sample
+### Descriptions MS TeamsApp resource type
```

Comment:
Not sure what does this title means. Could you please clarify or remove if not required?